notebooks/5. Compute sidewalk width.ipynb

```diff
@@ -131,7 +132,25 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "fb577bc3",
+   "id": "b3d9e7fe",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_points_on_line(line, max_seg_length): \n",
+    "    distance_delta = max_seg_length   \n",
```

Comment:
why do we have argument max_seg_length only to rename it to distance_delta?

Reply:
Indeed there's no point to this. Fixed it in the latest commit.